src/build_feature_store.py

```diff
@@ -746,20 +744,37 @@ def transform_building_features() -> DataFrame:
         .withColumn(
             "insulation_roof_r_value", extract_r_value(F.col("insulation_roof"))
         )
-        #  -- attached home transformations -- #
+        #  -- building type transformations -- #
         .withColumn(
             "is_attached",
-            F.col("geometry_building_type_acs") == "Single-Family Attached",
+            ~F.col("geometry_building_type_acs").isin(
+                ["Single-Family Detached", "Mobile Home"]
+            ),
+        )
+        .withColumn(
+            "is_mobile_home",
+            F.col("geometry_building_type_acs") == "Mobile Home",
+        )
+        .replace(
+            "None",
+            None,
+            subset=[
+                "geometry_building_number_units_sfa",
+                "geometry_building_number_units_mf",
+            ],
         )
         .withColumn(
             "n_building_units",
-            F.when(
-                F.col("geometry_building_number_units_sfa") == "None", F.lit(1)
-            ).otherwise(F.col("geometry_building_number_units_sfa")),
+            F.coalesce(
```

Comment:
Yes -- single family detatched and Mobile homes will have "None" for both columns, so we want to assign 1 here. Ill add a note here, but in general I chose to put most of the documentation [here](https://www.notion.so/rewiringamerica/Features-Upgrades-c8239f52a100427fbf445878663d7135?pvs=4#360863f318914aa4a1d7b8843e6cd24a) rather than in the code.